.travis.yml

```diff
@@ -9,6 +9,10 @@ otp_release:
   - 20.3
   - 20.0
   - 19.3
+env:
```

Comment:
I forgot to remove Travis CI from the project after adding GitHub Actions 😬 
I think this configuration is not needed.

Reply:
Ah, should I just remove `.travis.yml`?